book-1-orientation/chapters/STATIC_CLASSES.md

```diff
@@ -0,0 +1,106 @@
+# Static classes, methods and properties
+
+In C# all your code must be inside a `class`. Normally a class acts as a blueprint for creating objects within your program. However, there are times when you don't need to create a new object in order get your work done. Sometimes you just need a function or a property without the "context" of an object.
```

Comment:
all your code doesn't have to be inside a class necessarily: enums, structs, interfaces, etc.  probably pedantic and likely won't impact anything.

Reply:
I don't feel that mentioning static constructors is worth doing. They aren't used all that often anyway, plus from what I've seen so far, students have a hard time understanding regular constructors. I don't want to throw them a curve ball.

As for your other point about not all code having to be in a class... I was trying to lead into static classes by calling upon students experience so far, but in retrospect I don't think there's a good reason to bend the truth here I'll take it out.